lib/plugins/inventory.js

```diff
@@ -340,11 +340,13 @@ function inject (bot, { version, hideErrors }) {
       return
     }
     assert.ok(click.id <= actionId)
-    while (actionId > click.id) {
+    while (click !== undefined && actionId > click.id) {
       onAccepted()
       click = windowClickQueue.shift()
+       }
+    if (click !== undefined){
+        assert.ok(click)
```

Comment:
What’s the purpose of this assert? It’s hidden behind an != undefined

Reply:
yes i should remove it